.gitignore

```diff
@@ -114,3 +114,6 @@ env/
 
 ### Configuration files ###
 .env
+
+### Secrets files ###
+secrets.py
```

Comment:
This should be mentioned in the README. It should contain anything a developer needs to know to get the project running after cloning the repo.

But also, it shouldn't be necessary to have an ignored python file. Use the .env file instead to set environment variables and then load them in python code with `os.getenv('YOUR_ENV_VAR_NAME')`